js/jquery.imageplayer.js

```diff
@@ -59,22 +57,23 @@ if (typeof(jQuery) == 'undefined') alert('jQuery library was not found.');
             });
         });
 
-        function create_player() {
-            // Player elements.
-            player          = $('<div>').addClass('img_player');
+	function create_player() {
+            // Player elements.	
+	    player          = $('<div>').addClass('img_player');
             stage           = $('<div>').addClass('stage');
             controls        = $('<div>').addClass('controls');
             start           = $('<a>').attr('href', '#').addClass('start');
             prev            = $('<a>').attr('href', '#').addClass('prev');            
-            play_pause      = $('<a>').attr('href', '#');
+            play_pause      = $('<a id = "play_pause">').attr('href', '#');
             next            = $('<a>').attr('href', '#').addClass('next');
             end             = $('<a>').attr('href', '#').addClass('end');
-            scrubber        = $('<div>').addClass('scrubber');
-            scrubber_handle = $('<a>').attr('href', '#');
+            speed_up        = $('<a>').attr('href', '#').addClass('speed_up');
+            speed_down      = $('<a>').attr('href', '#').addClass('speed_down');
+            scrubber = $('<input id = "range" value="0" type="range" min="0" max="'+String(Number(settings.num_images) - 1)+'" role="input-range">').addClass('scrubber');
```

Comment:
The id is currently accessed to figure out the scrubber position in the handling functions. handle=document.getElementById('range');
handle.value = 0

So, I’m not sure how to go about accessing the input slider otherwise. Seems like other online documents point to using the .getElementById function: http://jsfiddle.net/girlie_mac/GKY2A

2. I have updated that on my end.

Reply:
Ooh ok I'm seeing your  last few comments, I will try to make this change and see how it goes.

Reply:
Perfect. Do not hesitate to ask for clarification or help I can provide.